Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -63,6 +63,40 @@ struct TextureMapperLayer::ComputeTransformData {
     }
 };
 
+class TextureMapperFlattenedLayer final : public TextureMapperPlatformLayer {
```

Comment:
TextureMapperFlattenedLayer doesn't need to inherit TextureMapperPlatformLayer.

Reply:
Fixed